### ISSUE TITLE
mos --help suggests updating even though current version is later 

### ISSUE THREAD

**Issue by sctv** [cesanta/mos-tool#1]
I have compiled from source, as I am trying it out on a rpi/arm but when I use `mos --help` it suggests I update it even though my compiled mos executable is at least 7 days older. 

Is it trying to downgrade to channel "latest"?   

Of course trying to upgrade it breaks it as it downloads an executable that is not compatible with arm.
  
`
mos --help
The Mongoose OS command line tool, v. 20180205-140203/master@e4dbea8a+.
Update channel: "latest". Checking updates... 
Out of date: new version available: 20180124-165446/master@f69e78ae+
Please run "mos update"
`

**Comment by dimonomid** [comment]
Yeah that's a known issue; the update check is very simple: it just compares the current mos build id with the one on the server (depending on the update channel: `release` or `latest`), and if they mismatch, it assumes that the server has a newer version.

Since you built your version from sources, your ID is of course different.

So this is a known issue, however there's no specific ETA for fixing it at the moment.

**Comment by sctv** [comment]
Thanks.

**Comment by sctv** [comment]
@dimonomid  should i close this?
